libnavigation-base/build.gradle

```diff
@@ -19,6 +19,10 @@ dependencies {
     api dependenciesList.mapboxSdkGeoJSON
 
     implementation dependenciesList.supportAnnotation
+    api dependenciesList.coroutinesCore
+    api dependenciesList.coroutinesAndroid
+
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.1.0'
```

Comment:
This is not necessary, it's duplicated ☝️ `api dependenciesList.coroutinesAndroid`

Reply:
👍